Mailchimp signup

ChefAndy opened the pull request:
Killed our internal mailing list sign-up functionality and pointed our mailing list sign-up form right at our mailchimp campaign.

ChefAndy committed:
pointed signup form at MC

jcushman commented on capstone/capapi/migrations/0016_delete_mailinglist.py:
```diff
@@ -0,0 +1,16 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```
Let's wait to delete the model until the rest of this lands, so we don't have to be too precise about the timing of copying the data over.

ChefAndy replied:
Ok.

jcushman commented on capstone/capweb/templates/index.html:
```diff
@@ -609,10 +614,15 @@ <h2 class="section-title pt-0 mb-3">
-                {% bootstrap_form subscribe_form %}
-                <div class="add">Add your email to keep in touch</div>
-                <button class="btn-secondary subscribe" type="submit">Subscribe!</button>
+          <form action="https://law.us3.list-manage.com/subscribe/post?u=4290964398813d739f2398db0&amp;id=e097736c6f"
```
Can we make the magic numbers here (and same elsewhere) into settings? That way dev/staging sites won't post to the real mailing list.

ChefAndy replied:
Sure.

jcushman reviewed (changes requested):
Thanks! Just a couple of tweaks.

ChefAndy committed:
replaced current mailing list functionality with mailchimp

* deleted:    capapi/tests/test_mailing_list.py
	* removed the mailing list functionality test
* modified:   capweb/forms.py
	* removed the signup form
* modified:   capweb/templates/index.html
	* changed the signup form to point to MC script
* modified:   capweb/urls.py
	* removed the mailing list routes
* modified:   capweb/views.py
	* changed the index view to get magic values from settings, and remove form info
* modified:   config/settings/settings_base.py
	* added blank value for MC magic values
* modified:   config/settings/settings_prod.py
	* added actual MC magic values

jcushman commented:
Thanks!
